### PR TITLE
adds grey hint colour, and grey background to dropdowns

### PR DESCRIPTION
**Pull request by leejordan**

### CODE REVIEW

File: less/modules/navigation.less [hunk 1]
```diff
@@ -201,8 +201,7 @@
                 z-index: (@z-index-top + 1);
                 color: @colour-brand;
                 background-color: @colour-white;
-                border: @border-default;
-                //border-bottom: none;
+                border: 1px solid @colour-grey-lightest;
```
Comment (notlee) [inline less/modules/navigation.less]: If we have a bunch of borders not using `@border-colour`, should tidy up and get rid of the alias?

Comment (leejordan) [inline less/modules/navigation.less]: I considered doing what you suggest but decided against it based on the fact that this is the only exception to the rule so far.

File: less/modules/page-sections.less [hunk 2]
```diff
@@ -35,6 +35,10 @@
     background-color: @colour-grey-lightest;
 }
 
+.page-section--hint {
```
Comment (notlee) [inline less/modules/page-sections.less]: Add this in the `/patterns/page-sections` docs?

Comment (leejordan) [inline less/modules/page-sections.less]: agreed, and done

**Pull request merged**